frontend/freebase2wikidata.js

```diff
@@ -260,7 +260,56 @@ $(function() {
     var title = mw.config.get('wgTitle');
     return qidRegEx.test(title) ? title : false;
   }
+  
+ function generateNav() {
```

Comment:
I don't get where the "right of the text" should be, can you provide a simple sketch or screenshot?